sdk/src/cose_validator.rs

```diff
@@ -1017,25 +1082,25 @@ pub mod tests {
     #[test]
     fn test_verify_ec_good() {
         // EC signatures
-        let mut validator = get_validator("es384").unwrap();
+        // let validator = get_validator("es384").unwrap();
 
-        let sig_es384_bytes = include_bytes!("../tests/fixtures/sig_es384.data");
-        let data_es384_bytes = include_bytes!("../tests/fixtures/data_es384.data");
-        let key_es384_bytes = include_bytes!("../tests/fixtures/key_es384.data");
+        // let sig_es384_bytes = include_bytes!("../tests/fixtures/sig_es384.data");
```

Comment:
This one is a little mystery - I can't have these tests pass using the Ring backent, and I can't seem to be able to validate them with command line openssl (`openssl dgst -sha256 -verify ../tests/fixtures/key_es384.data -signature ../tests/fixtures/sig_es384.data ../tests/fixtures/data_es384.data`).

My generated tests, otoh, work ok with both Openssl and Ring validation.

Left commented version to flag the oddity.

Reply:
Do you handle IEEE P1363 signature format?  That is what is spec'd by C2PA

Reply:
Complete oversight on my part. Should be possible to re-add it, inasmuch as the packing/unpacking seems to happens outside of openssl (seems to be the case?).

Reply:
IEEE P1363 managemen added